try:
    print(5 / 0)
except:
    print('There was an error')
x = 56
print(x + 100)
raise Exception('Bli bla blo')